openqabot/pc_helper.py

```diff
@@ -130,6 +130,6 @@ def is_newer(date1: str, date2: str) -> bool:
         if (region is not None) and (region != image["region"]):
             continue
         # Get latest one based on 'publishedon'
-        if recentimage is None or is_newer(image["publishedon"], recentimage["publishedon"]):
+        if recentimage is None or is_newer(image["publishedon"], recentimage["publishedon"]):  # pragma: no branch
```

Comment:
To me this looks like a branch. (Actually, there are multiple branches due to the `or`.) So why `no branch`?

Reply:
And what makes this branch special then (so we don't need to track branch coverage even though we might do that generally)?